release-content/0.14/migration-guides/12234_Move_commands_module_into_bevyecsworld.md

```diff
@@ -1,8 +1,9 @@
-`Command` and `CommandQueue` have been moved from `bevy::ecs::system` to `bevy::ecs::world`.
+`Command` and `CommandQueue` have been moved from `bevy::ecs::system` to `bevy::ecs::world`. If you import them directly, you will need to update your import statements. (This does not affect you if you just import the prelude.)
 
 ```rust
-// 0.13
+// Before
```

Comment:
Previous guides have generally used
```
// 0.13
// 0.14
```
It was a bit inconsistent early on but very consistent in recent guides.

Reply:
Ok, let me fix this real quick. (I'll open another PR to fix all of my past migration guides too.)